src/helpers/updateNftAnalytics.ts

```diff
@@ -0,0 +1,37 @@
+import { Address, BigInt, log } from '@graphprotocol/graph-ts';
+
+import {
+  NftAnalytics,
+} from '../../generated/schema';
+
+import { ONE, ZERO } from './common';
+
+import { nftId } from './idTemplates';
+
+export function updateNftAnalytics(
+    contractAddress: Address,
+    tokenId: BigInt,
+    tokenSold: boolean,
+    royaltiesClaimed: BigInt
+): void {
+
+    const id = nftId(contractAddress.toHex(), tokenId.toString());
+    let _nftAnalytics = NftAnalytics.load(id);
+    
+    if (!_nftAnalytics) {
+        _nftAnalytics = new NftAnalytics(id);
+        _nftAnalytics.contractAddress = contractAddress;
+        _nftAnalytics.tokenId = tokenId;
+        _nftAnalytics.totalSales = ZERO;
+        _nftAnalytics.totalRoyalties = ZERO;
+    }
+    if (tokenSold === true) {
+        _nftAnalytics.totalSales.plus(ONE);
```

Comment:
This does not assign the value, it needs to be:
`_nftAnalytics.totalSales = _nftAnalytics.totalSales.plus(ONE)`

Reply:
Fixed